Nukeeper.BitBucketLocal/NuKeeper.BitBucketLocal.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
If these are needed, would be best to remove the condition also please add

```
 <PropertyGroup>
        <CodeAnalysisRuleSet>..\CodeAnalysisRules.ruleset</CodeAnalysisRuleSet>
    </PropertyGroup>
    ```

Reply:
The condition and noWarn is a copy from the BitBucket project. 
I've added the <CodeAnalysisRuleSet>

Reply:
Actually I cannot build in releasemode if I do not add:  
 `  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
    <NoWarn>1701;1702;CA1051;CA1707;CA2227;CA1056;</NoWarn>
  </PropertyGroup>`
I've added the CodeAnalysisRuleSet part

Reply:
Instead of the 2 PropertyGroups you should be able to just add this, It will then apply to all release types:
```
<PropertyGroup> 
    <NoWarn>1701;1702;CA1051;CA1707;CA2227;CA1056;</NoWarn> 
</PropertyGroup>
```

Reply:
I'll change that right away